index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="tool-box">
+        <div class="headder">
+            <h4 class="name">Welcome</h4>
+            <img class="icon" src="images/Union.png" alt="icon">
+        </div>
+    </section>
+    <!-- welcome_page -->
+    <section class="welcome_page">
+        <!-- welcome message -->
+        <div class="welcome_message">
+            <h1 class="greeting">Hey there. I'm Bobby</h1>
```

Comment:
[ Required ] Please consider changing the text `I'm Bobby` to your name, since this is going to be your portfolio for some time it will be better if you customize to use your name.